python/bridgestan/model.py

```diff
@@ -165,6 +186,18 @@ def __init__(
         self._param_unc_num.restype = ctypes.c_int
         self._param_unc_num.argtypes = [ctypes.c_void_p]
 
+        num_params = self._param_unc_num(self.model)
+        unc_p_double_array = array_ptr(
+            dtype=ctypes.c_double,
+            flags=("C_CONTIGUOUS", "WRITEABLE"),
```

Comment:
If I'm reading this right, it's great that you got out variables into Python code.